### PR TITLE
Adding pcp-mmvstatsd 0.1 RPM

### PR DESCRIPTION
**Pull request by lzap**
This is an alternative to https://github.com/theforeman/foreman-packaging/pull/2429

Please do not merge until I properly test this.

* [x] Nightly

### CODE REVIEW

**Comment by lzap** [comment]
@ekohl hey any hint why CI is broken?

**Comment by ekohl** [comment]
@lzap you're missing the entry in `package_manifest.yaml`

**Comment by lzap** [comment]
Rebased that should do it.

**Comment by ekohl** [comment]
[test rpm]

File: package_manifest.yaml [hunk 1]
```diff
@@ -126,6 +126,7 @@ foreman_packages:
     nodejs-webpack-stats-plugin: {}
     nodejs-webpack: {}
     nodejs-zxcvbn: {}
+    pcp-mmvstatsd: {}
```
Comment (ekohl) [inline package_manifest.yaml]: The build fails because this doesn't match the package name.

**Comment by lzap** [comment]
This is now ready @ekohl thanks.

If there is a chance to get this into downstream for tomorrow that's be awesome @evgeni - https://bugzilla.redhat.com/show_bug.cgi?id=1578693

Ping me on IRC if something blocks you.

File: package_manifest.yaml (hunk 1 above)
Comment (lzap) [reply package_manifest.yaml]: This is now fixed.

File: packages/foreman/pcp-mmvstatsd/pcp-mmvstatsd.spec [hunk 2]
```diff
@@ -0,0 +1,67 @@
+%define debug_package %{nil}
+
+Name:    pcp-mmvstatsd
+Version: 0.2
+Release: 1%{?dist}
+Summary: Statsd to PCP MMV aggregator
+License: MIT and BSD
+URL:     https://github.com/lzap/%{name}
+
+Source0: https://codeload.github.com/lzap/%{name}/tar.gz/%{version}
```
[the remark this answers is not in the document]
Comment (lzap) [reply packages/foreman/pcp-mmvstatsd/pcp-mmvstatsd.spec]: OOOOOOOOOH! Finally a solution :-) Rebased.

**Comment by evgeni** [comment]
thanks @lzap!

**Comment by evgeni** [comment]
``
koji add-pkg foreman-nightly-nonscl-rhel7 pcp-mmvstatsd --owner=kojiadm
```

http://koji.katello.org/koji/taskinfo?taskID=98645

**Comment by lzap** [comment]
Thanks a bunch folks!

**Pull request merged**